crates/console-timer/src/lib.rs

```diff
@@ -0,0 +1,75 @@
+/*!
+
+Measuring invocation time on the Web can be done with `console.time`
+and `console.timeEnd`.
+
+This API wraps both the `time` and `timeEnd` calls into a single type
+named `ConsoleTimer`, ensuring both are called.
+
+## Scoped measurement
```

Comment:
Can we also have a section on the RAII-style usage below this? Potentially showing timing asynchronous operations, where the scoped callbacks approach can't be used.